src/workspace-data/data-table/wds/WdsTroubleshooter.ts

```diff
@@ -83,7 +83,12 @@ export const WdsTroubleshooter = ({ onDismiss, workspaceId, mrgId }) => {
   const troubleShooterText: [string, string | null, boolean, boolean, ReactNode?][] = [
     ['Workspace Id', workspaceId, false, !!workspaceId],
     ['Resource Group Id', mrgId, false, !!mrgId],
-    ['App listing', `${numApps} app(s) total`, numApps == null, !!numApps && numApps !== 'unknown'],
+    [
+      'App listing',
+      numApps === null ? null : `${numApps} app(s) total`,
+      numApps == null,
```

Comment:
Not sure I understand this chain of logic...what's the `numApps == null` clause for?

Reply:
`numApps` is null while the apps list is being loaded from Leo. Currently, this results in the troubleshooter showing "null apps(s) total" during that time. This changes it to render nothing (null) in that row while `numApps` is loading.

Reply:
Oh, I misunderstood the question. That's the logic for determining if the App listing row is still loading.
https://github.com/DataBiosphere/terra-ui/blob/d1438feef5b2739b9d95dc34a8cbe2ac2b04c096/src/workspace-data/data-table/wds/WdsTroubleshooter.ts#L76-L83